moonraker/components/paneldue.py

```diff
@@ -760,6 +773,44 @@ def _run_paneldue_M20(self, arg_p: str, arg_s: int = 0) -> None:
                 response['files'] = flist
         self.write_response(response)
 
+    def get_macro_dir(self, path: str, issubdir: boolean):
```

Comment:
The `boolean` annotation is incorrect here, it should be `bool`.  The return value also needs to be annotated.